WORKSPACE

```diff
@@ -422,6 +422,8 @@ http_archive(
         "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/releases/download/3.1.0/bazel-toolchains-3.1.0.tar.gz",
         "https://github.com/bazelbuild/bazel-toolchains/releases/download/3.1.0/bazel-toolchains-3.1.0.tar.gz",
     ],
+    patches = ["@//third_party:bazel_toolchains/0001-Rename-target_compatible_with-to-internal_target_com.patch"],
```

Comment:
FYI I have no idea how the `patches` stuff works.  Do you need someone else to look at this part or are you confident in it?

Reply:
@philwo do you know the right way to review core `WORKSPACE` changes?

Reply:
@gregestren I think we treat them just like any other code, there's nothing really special about the WORKSPACE file. A temporary patch like this seems fine to me. We would eventually like this to be merged upstream, but.. uhm.. it seems bazel-toolchains is not exactly in a well-maintained state at the moment. 😬

Reply:
I'll leave this as-is for now and bazel-toolchains will eventually catch up :)

Reply:
Merge update: `third_party` changes have to be in their own dedicated commit. Can we move the `.patch` file to its own commit and get that submitted first?